### DISCUSSION TITLE
初次启动时会有注册表报错

### DISCUSSION THREAD

**Discussion by zhaowenjunan** [Lunova-Studio/DarkMode2#18] [Q&A]
初始化注册表时发生了不可逆的错误

System.Net.WebException: 操作超时
在 System.Net.HttpWebRequest.GetRequestStream(TransportContext&context)
在 System.Net.HttpWebRequest.GetRequestStream()
在 DarkMode 2.Views.MainWindow.SendPostRequest(String url, String data) 位置 E:\ProJect\C#\DarkMode 2\Views\MainWindow.xaml.cs: 行号179
在 DarkMode 2.Views.MainWindow..ctor(MainWindowViewModeIviewModeI, IPageSetvice pageSetvice, ITaskBarService taskBarService,lTestWindowService testWindowService IThemeService themeServices) 位置 E:\ProJect\C#\DarkMode 2\Views\MainWindow.xaml.cs:行号 137

**Comment by bao-cn** [comment]
请提供详细的信息，操作系统、操作系统版本、软件版本等，请尝试以管理员模式重新运行，如果还是无法解决请按照首页的方法清除注册表，再使用管理员模式运行程序，注意，不建议将软件放置在C盘目录下运行。

**Reply by zhaowenjunan** [reply]
感谢回复，我这边有且仅在初次启动时才出现这个报错，后续重启、使用过程中无异常，问题无法复现。
我下班回家后给您系统信息吧～
